AmplifyPlugins/API/AWSAPICategoryPlugin/Interceptor/SubscriptionInterceptor/IAMAuthInterceptor.swift

```diff
@@ -11,17 +11,17 @@ import AWSPluginsCore
 import Amplify
 import AppSyncRealTimeClient
 
-class IAMAuthInterceptor: AuthInterceptor {
+public class IAMAuthInterceptor: AuthInterceptor {
```

Comment:
Do we need this to be public?

Reply:
nice catch, it was copied over from the shared package so it was public, making internal